model/EligibilityService.php

```diff
@@ -527,6 +541,34 @@ public function deletedTestTaker($event)
         }
     }
 
+    /**
+     * @param Event $event
+     * @throws \common_exception_InconsistentData
+     */
+    public function deleteDelivery(Event $event)
```

Comment:
I think it would be better to move this method to dedicated event listener

Reply:
DeliveryListener created